phalcon/Mvc/Model.zep

```diff
@@ -3900,6 +3900,16 @@ abstract class Model extends AbstractInjectionAware implements EntityInterface,
                                         );
                                     }
 
+                                    /**
+                                     * Get actual values before comparison
+                                     */
+                                    if is_object(snapshotValue) && snapshotValue instanceof RawValue {
```

Comment:
```suggestion
                                    if snapshotValue instanceof RawValue {
```

Reply:
`instanceof` expects an object instance, but `snapshotValue` can be other type